lib/facility_maker.rb

```diff
@@ -0,0 +1,50 @@
+class FacilityMaker
+  attr_reader :add_facility, :facility, :add_facility, :facility_info, :name
+  attr_accessor :name, :address, :phone, :facilities, :add_facility
+
+  def new_facility(state_facilities)
+    state_facilities.map do |state_facility|
+      gen_facility = Facility.new({name: new_facility_name(state_facility), address: new_facility_address(state_facility), 
+      city: state_facility[:city], state: state_facility[:state], zipcode: new_facility_zipcode(state_facility), 
+      phone: new_facility_phone(state_facility)})
+    end
+  end
+
+  def new_facility_name(state_facility)
+    if state_facility[:website] && state_facility[:website].include?("oregon")
+      state_facility[:title]
+    elsif state_facility[:state] && state_facility[:state] == "NY"
+      state_facility[:office_name]
+    else state_facility[:state] && state_facility[:state] == "MO"
```

Comment:
```suggestion
    elsif state_facility[:state] && state_facility[:state] == "MO"
```
This should be an `elsif` not an `else`. Ruby doesn't evaluate anything to the right of an `else`. Else is catch all - none of the previous conditions were met so we do something by default. If we care about conditions, we should use an `elsif`. Same for lines 28, 36, and 46.